pkg/features/kube_features.go

```diff
@@ -411,13 +411,13 @@ const (
 	CSIMigrationAzureFileComplete featuregate.Feature = "CSIMigrationAzureFileComplete"
 
 	// owner: @divyenpatel
-	// alpha: v1.19
+	// beta: v1.19 (vSphere vCenter/ESXi Version: 7.0u1, HW Version: VM version 15)
```

Comment:
Do you have an issue or PR with #sig-cloud-provider on the deprecation plan for vSphere 7.0.0 and below?

Reply:
> 2. set default storage class to CSI driver

The point of CSI migration is you don't need to change your default storageclass

Reply:
> > 1. set default storage class to CSI driver
> 
> The point of CSI migration is you don't need to change your default storageclass

LOL you are right. It's been a long day.

Reply:
>  Do you have an issue or PR with #sig-cloud-provider on the deprecation plan for vSphere 7.0.0 and below?

@misterikkit let me get help from @andrewsykim regarding this.

Reply:
should the string say - `requires: vSphere xxx`

Reply:
done @gnufied Can you add back `lgtm`.